FightPandemics/Features/Post/CreatePost/PostCreationTagView.swift

```diff
@@ -0,0 +1,189 @@
+//
+//  PostCreationTagView.swift
+//  FightPandemics
+//
+//  Created by Onyekachi Ezeoke on 17/05/2020.
+//
+//  Copyright (c) 2020 FightPandemics
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class PostCreationTagView: UIView {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+    }
+
+    private struct Dimension {
+        static let buttonHeight: CGFloat = 44.2
+        static let stackViewSpacing: CGFloat = 19
+    }
+
+    let mockTags = ["Medical supplies", "Groceries/Food", "Business", "Education", "Legal", "Wellbeing/Mental", "University", "Entertainment", "Information", "Funding", "R&D", "Tech", "Others"]
+
+    private var tagCount: Int? {
+        didSet {
+            if let tagCount = tagCount {
+                let localizedString = String.localizedStringWithFormat(stringFormat, String(tagCount))
+                addTagButton.setTitle(localizedString, for: .normal)
+            }
+        }
+    }
+
+    private var stringFormat: String {
+        let tagCount = self.tagCount ?? 0
+        if tagCount == 0 {
+            return "Tag-Button-With-Unselected-Tag".localized
+        } else if tagCount == 1 {
+            return "Tag-Btn-With-One-Selected-Tag".localized
+        } else {
+            return "Tag-Btn-With-Many-Selected-Tags".localized
+        }
+    }
+
+    private lazy var addTagButton: UIButton = {
+        let button = UIButton()
+        let localizedString = String.localizedStringWithFormat(stringFormat, 0)
```

Comment:
consider using the `func localized(formattedWith:)` utility in `String+FightPandemics`. not required! just an attempt to make something like this less verbose

Reply:
Okay, thanks 👍🏻